src/main/cpp/chassis/swerve/headingStates/FaceGoalHeading.h

```diff
@@ -30,6 +30,5 @@ class FaceGoalHeading : public ISwerveDriveOrientation
 
 private:
     DragonTargetFinder m_targetFinder;
-
-    DragonCamera *m_vision;
```

Comment:
This class has either been deleted or will be deleted

Reply:
Fixed with merge